Bixal/move.mil
# Triage react app versioning and paths

In react apps, updates the `webpack-dev-server` version to allow `npm start` to run properly, moves it to devDependencies to avoid vulnerabilities in prod environment, and updates the package-lock.json accordingly. Also fixes a bug in the react app that had made it to the staging site but not yet to prod, and was preventing the weight estimator from working properly. If successful, all four apps should work properly in local Drupal (and eventually on staging & prod).

No JIRA ticket (yet?) for this one.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- See summary above

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Go to `web/modules/custom/react_tools/tools`
1. Run `npm install`
1. Run `npm start` and load EACH of the four apps locally. They should all at least load initially.
1. Go to EACH of the four apps in Drupal and make sure they work properly, including results behavior.

## Screenshots

N/A
